src/main/js/app.js

```diff
@@ -64,7 +64,6 @@ if (isProduction) {
       }
     });
 } else {
-  window.Perf = require("react-addons-perf");
   require("../../tests/runTests");
```

Comment:
Perhaps we should think about running these tests with `mocha` or `jest`. Could be done in a future branch/task. We could than leverage from automated test execution on build time.